features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Overview.jsx

```diff
@@ -664,7 +664,31 @@ function Overview() {
                             </Box>
                         </>
                     )}
-
+                    {api.additionalProperties && Object.keys(api.additionalProperties).length > 0 && (
+                        <>
+                            <Box mt={6}>
+                                <Typography variant='subtitle2' className={classes.sectionTitle}>
+                                    <FormattedMessage
+                                        id='Apis.Details.Overview.additional.properties'
+                                        defaultMessage='Additonal properties'
+                                    />
+                                </Typography>
+                            </Box>
+                            <Box mt={1} ml={1}>
+                                {Object.keys(api.additionalProperties).map((property) => {
```

Comment:
It's better and safe to iterate using Object entries 
I:e
```js
Object.entries(obj).forEach(([key, value]) => console.log(`${key}: ${value}`));
```
[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)

Indexing `additionalProperties[property]` with the key would fail if the object prototype contains additional properties

Reply:
Fixed. https://github.com/wso2/carbon-apimgt/pull/10326/commits/38aa29713cc2df3b1b0d27d550d442692e46eb8f